app/models/bookable_slot.rb

```diff
@@ -23,7 +24,14 @@ def self.without_appointments
               appointments.end_at = #{quoted_table_name}.end_at
             SQL
          )
-      .where('appointments.start_at IS NULL')
+      .where(<<-SQL
```

Comment:
didn't work, and was struggling to get it to play ball with the or method (and having a join)